meshroom/ui/reconstruction.py

```diff
@@ -442,8 +449,12 @@ def reloadPlugins(self):
             for node in plugin.nodes.values():
                 if node.reload():
                     nodeTypes.append(node.nodeDescriptor.__name__)
+        self.pluginsReloaded.emit(nodeTypes)
 
+    @Slot(list)
+    def _onPluginsReloaded(self, nodeTypes: list):
         self._graph.reloadNodePlugins(nodeTypes)
+        self.parent().showMessage("Plugins reloaded !", "ok")
```

Comment:
```suggestion
        self.parent().showMessage("Plugins reloaded!", "ok")
```
to respect the syntax in English.